torchao/quantization/quant_api.py

```diff
@@ -501,13 +522,18 @@ def apply_int8_dynamic_activation_int4_weight_quant(weight, group_size=32, mappi
     quant_max = 7
 
     # input settings
-    input_quant_func = _int8_asymm_per_token_quant
+    if input_mapping_type == MappingType.ASYMMETRIC:
+        input_quant_func = _int8_asymm_per_token_quant
+    elif input_mapping_type == MappingType.SYMMETRIC:
+        input_quant_func = _int8_symm_per_token_quant
+    else:
+        assert False, f"Unsupported input mapping type: {input_mapping_type}"
 
-    weight = to_affine_quantized_intx(weight, mapping_type, block_size, target_dtype, quant_min, quant_max, eps)
+    weight = to_affine_quantized_intx(weight, mapping_type, block_size, target_dtype, quant_min, quant_max, eps, _layout=layout)
     weight = to_linear_activation_quantized(weight, input_quant_func)
     return weight
 
-def int8_dynamic_activation_int4_weight(group_size=32, mapping_type=MappingType.SYMMETRIC):
+def int8_dynamic_activation_int4_weight(group_size=32, layout=PlainLayout(), mapping_type=MappingType.SYMMETRIC, input_mapping_type=MappingType.ASYMMETRIC):
```

Comment:
can you update the doc block, maybe with some description of how the supported layouts